### DISCUSSION TITLE
jBang Support

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [raxod502/riju#117] [Ideas]
@raxod502 congrats on Launch of your open source￼ project. I think it would be great to have a JBang support instead of Java. JBang is very powerful and has more features than standalone Java jdk. https://github.com/jbangdev/jbang

Cc @maxandersen

**Comment by maxandersen** [comment]
thanks for cc'ing me.

I made a `jbang` pull request at #118 that adds jbang but I think it would make more sense to enhance the java to use jbang to do the building/running to get access to the full java ecosystem.

**Comment by shresthaujjwal** [comment]
wow @maxandersen that was a quick turn around, thank you appreciate it. I was planning on using this tool for my internal project with jbang, love both the project. Great collaboration. Thanks again.
